pytorch_lightning/loggers/neptune.py

```diff
@@ -276,118 +371,50 @@ def save_dir(self) -> Optional[str]:
 
     @property
     def name(self) -> str:
-        if self.offline_mode:
+        try:
+            self.run.sync()
+        except NeptuneOfflineModeFetchException:
             return 'offline-name'
-        else:
-            return self.experiment.name
+        return self.run['sys/name'].fetch()
 
     @property
     def version(self) -> str:
-        if self.offline_mode:
+        try:
+            self.run.sync()
+        except NeptuneOfflineModeFetchException:
             return 'offline-id-1234'
-        else:
-            return self.experiment.id
-
-    @rank_zero_only
-    def log_metric(
-        self, metric_name: str, metric_value: Union[torch.Tensor, float, str], step: Optional[int] = None
-    ) -> None:
-        """
-        Log metrics (numeric values) in Neptune experiments.
-
-        Args:
-            metric_name: The name of log, i.e. mse, loss, accuracy.
-            metric_value: The value of the log (data-point).
-            step: Step number at which the metrics should be recorded, must be strictly increasing
-        """
-        if is_tensor(metric_value):
-            metric_value = metric_value.cpu().detach()
+        return self.run['sys/id'].fetch()
 
-        if step is None:
-            self.experiment.log_metric(metric_name, metric_value)
-        else:
-            self.experiment.log_metric(metric_name, x=step, y=metric_value)
+    def _raise_deprecated_api_usage(self, f_name, sample_code):
+        raise ValueError(f"Function you've used  is deprecated."
+                         f" Instead of `logger.{f_name}` you can use:\n"
+                         f"\tfrom neptune.new.attributes import constants"
+                         f"\t{sample_code}")
 
     @rank_zero_only
-    def log_text(self, log_name: str, text: str, step: Optional[int] = None) -> None:
-        """
-        Log text data in Neptune experiments.
-
-        Args:
-            log_name: The name of log, i.e. mse, my_text_data, timing_info.
-            text: The value of the log (data-point).
-            step: Step number at which the metrics should be recorded, must be strictly increasing
-        """
-        self.experiment.log_text(log_name, text, step=step)
+    def log_metric(self, *args, **kwargs):
+        self._raise_deprecated_api_usage("log_metric", "logger.run[constants.LOG_ATTRIBUTE_SPACE]['key'].log(42)")
```

Comment:
Do we want the user to see `constants.LOG_ATTRIBUTE_SPACE`? Or just `logger.run['log/key'].log(42)`